torch/csrc/distributed/c10d/init.cpp

```diff
@@ -700,6 +701,13 @@ They are used in specifying strategies for reduction collectives, e.g.,
           "wait",
           &::c10d::ProcessGroup::Work::wait,
           py::arg("timeout") = kNoTimeout,
+          py::call_guard<py::gil_scoped_release>())
+      .def(
+          "get_future",
```

Comment:
Add some docs for this method since it is a public method. Example: https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/rpc/init.cpp#L291

Reply:
Should we make it private like `_get_future()` as well?